File: Registro de Alunos/app/src/main/java/com/example/artur/registrodealunos/Modelo/Aluno.java
package com.example.artur.registrodealunos.Modelo;

import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;

@Entity
public class Aluno {

    @Id Long id;
    private String nome;
    private String curso;
    private Faculdade faculdade;

    Aluno(){ }


    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getCurso() {
        return curso;
    }

    public void setCurso(String curso) {
        this.curso = curso;
    }

    public Faculdade getFaculdade() {
        return faculdade;
    }

    public void setFaculdade(Faculdade faculdade) {
        this.faculdade = faculdade;
    }
}

File: Registro de Alunos/app/src/main/java/com/example/artur/registrodealunos/Modelo/Aluno.kt
package com.example.artur.registrodealunos.Modelo

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id
import io.objectbox.relation.ToOne

@Entity
data class Aluno (var nome: String,
                  var curso: String){
    @Id var id: Long = 0
    lateinit var faculdade: ToOne<Faculdade>
}